test/Unused/Parser/InternalSpec.hs

```diff
@@ -0,0 +1,21 @@
+module Unused.Parser.InternalSpec where
+
+import Test.Hspec
+import Unused.Types
+import Unused.Parser.Internal
+
+main :: IO ()
+main = hspec spec
+
+spec :: Spec
+spec = parallel $
```

Comment:
Given the test suite is already pretty fast, it's hard to tell how much value it's providing, but I'll take it anyway!